templates/base.html

```diff
@@ -102,6 +106,33 @@
         </script>
         {% endblock script %}
         {% include '_includes/stat_counter.html' %}
+
+        {% if (article and article.photo_gallery) %}
+        <!-- 'photo' Gallery support -->
+        <!-- Magnific Popup core JS file -->
+        <script src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/magnific/magnific-popup.js"></script>
```

Comment:
lightbox support should be optional. We can enable it by default.

1. variable in `_defaults.html` set to ture
1. If it is true then show include`lightbox` script and js
1. If is not true then show vanilla gallery from your previous PR

Reply:
solved in latest push